housingResidentInformationAPI/serverless.yml

```diff
@@ -1,4 +1,4 @@
-service: uh-resident-information-api
+service: housing-resident-information-api
```

Comment:
Changing the name of this service will mean that a new aws stack will be created, so after it has been fully merged into production the old one can be removed from aws and where we have the base URL for staging and production saved in the parameter store for single view, these will need to be changed to the new URLs